notebooks/data-sources/gcsweb-ci/prow_archive_discovery.ipynb

```diff
@@ -92,7 +92,7 @@
     {
```

Comment:
<p>So IIUC the issue was that build id <code>1364869749170769920</code> doesn't exist anymore and that's why running this cell gave an error. Can we reliably say build id <code>1380606326060617728</code> will exist on GCS couple of months from now? Basically, is there a way to <em>not</em> hardcode the build id and use something like <code>sorted(build_events_data.keys())[0]</code>?  WDYT?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>+1</p>

Reply:
Made changes according to the feedback. Thanks :) @chauhankaranraj.